internal/response.go

```diff
@@ -91,10 +91,13 @@ func (s *SPOA) processResponse(msg spoe.Message) ([]spoe.Action, error) {
 			if !ok {
 				return nil, fmt.Errorf("invalid argument for http response body, []byte expected, got %v", arg.Value)
 			}
-			_, err := tx.ResponseBodyWriter().Write(body)
+			it, _, err := tx.WriteRequestBody(body)
```

Comment:
@jcchavezs thanks for the project! I am looking into it to do something similar to nginx and ingress nginx :)

While copying (eh, sorry!) this part of the code, I've realized this is WriteRequestBody, but shouldn't it be WriteResponseBody as we are on the response part?

Reply:
Fixed in #68 ([spoa.go#L375](https://github.com/corazawaf/coraza-spoa/blob/57efefa1a3f0e9086ac00198849d97bad5c1b99a/internal/spoa.go#L375))